packages/drivers/routerlicious-driver/src/test/errorUtils.spec.ts

```diff
@@ -229,11 +240,14 @@ describe("ErrorUtils", () => {
 				},
 				handler,
 			);
-			assertExpectedMessage(error.message);
-			assert.strictEqual(error.errorType, DriverErrorType.throttlingError);
-			assert.strictEqual(error.canRetry, true);
-			assert.strictEqual(error.retryAfterSeconds, 5);
-			assert.strictEqual((error as any).statusCode, 429);
+
+			if (isIThrottlingWarning(error)) {
```

Comment:
This will result in no asserts being called if the error _isn't_ `IThrottlingWarning`. Rather than an `if`, we need to make this another `assert` call. I.e.
```typescript
assert(isIThrottlingWarning(error));
```

I _believe_ that should do the type-narrowing we need. If not, you can cast to the interface afterwards.